dashboard/app/models/backpack.rb

```diff
@@ -13,4 +13,23 @@
 #  index_backpacks_on_user_id  (user_id)
 #
 class Backpack < ApplicationRecord
+  belongs_to :user
+  validates_uniqueness_of :storage_app_id
+  validates_uniqueness_of :user_id
```

Comment:
Is it worth changing the index to be unique?

Reply:
I intended to do that [here](https://github.com/code-dot-org/code-dot-org/pull/41635), but looking at the generated schema file it seems I used the wrong syntax. I'll do a follow-up to actually make it unique and remove this here